### ISSUE TITLE
Exception thrown during creation node interface via API

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-server#77]
Linstor version `1.0.1`

Steps to reproduce:

```
curl -X POST -H "Content-Type: application/json" -d '{"net_interfaces": [{"satellite_port": 3366, "satellite_encryption_type": "Plain", "name": "default", "address": "1.2.3.4"}], "type": "Satellite", "name": "test"}' http://localhost:3370/v1/nodes
curl -X POST -H "Content-Type: application/json" -d '{"name": "data", "address": "4.3.2.1"}' http://localhost:3370/v1/nodes/test/net-interfaces
```

```
ERROR REPORT 5D83A631-00000-000004

============================================================

Application:                        LINBIT? LINSTOR
Module:                             Controller
Version:                            1.0.1
Build ID:                           98a9905de43631b745c7c0741c2ef8f577513b23
Build time:                         2019-08-09T07:02:30+00:00
Error time:                         2019-09-20 14:39:58
Node:                               linstor-controller-0

============================================================

Reported error:
===============

Category:                           RuntimeException
Class name:                         NullPointerException
Class canonical name:               java.lang.NullPointerException
Generated at:                       Method 'lambda$createNetInterface$7', Source file 'Nodes.java', Line #300


Call backtrace:

    Method                                   Native Class:Line number
    lambda$createNetInterface$7              N      com.linbit.linstor.api.rest.v1.Nodes:300
    doInScope                                N      com.linbit.linstor.api.rest.v1.RequestHelper:226
    doInScope                                N      com.linbit.linstor.api.rest.v1.RequestHelper:199
    createNetInterface                       N      com.linbit.linstor.api.rest.v1.Nodes:289
    invoke0                                  Y      jdk.internal.reflect.NativeMethodAccessorImpl:unknown
    invoke                                   N      jdk.internal.reflect.NativeMethodAccessorImpl:62
    invoke                                   N      jdk.internal.reflect.DelegatingMethodAccessorImpl:43
    invoke                                   N      java.lang.reflect.Method:566
    lambda$static$0                          N      org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory:52
    run                                      N      org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1:124
    invoke                                   N      org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher:167
    doDispatch                               N      org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker:176
    dispatch                                 N      org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher:79
    invoke                                   N      org.glassfish.jersey.server.model.ResourceMethodInvoker:469
    apply                                    N      org.glassfish.jersey.server.model.ResourceMethodInvoker:391
    apply                                    N      org.glassfish.jersey.server.model.ResourceMethodInvoker:80
    run                                      N      org.glassfish.jersey.server.ServerRuntime$1:253
    call                                     N      org.glassfish.jersey.internal.Errors$1:248
    call                                     N      org.glassfish.jersey.internal.Errors$1:244
    process                                  N      org.glassfish.jersey.internal.Errors:292
    process                                  N      org.glassfish.jersey.internal.Errors:274
    process                                  N      org.glassfish.jersey.internal.Errors:244
    runInScope                               N      org.glassfish.jersey.process.internal.RequestScope:265
    process                                  N      org.glassfish.jersey.server.ServerRuntime:232
    handle                                   N      org.glassfish.jersey.server.ApplicationHandler:680
    service                                  N      org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer:353
    run                                      N      org.glassfish.grizzly.http.server.HttpHandler$1:200
    doWork                                   N      org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker:569
    run                                      N      org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker:549
    run                                      N      java.lang.Thread:834


END OF ERROR REPORT.

```

**Comment by ghernadi** [comment]
That bug was fixed shortly after releasing `1.0.1`. Please update, the bug is already fixed in newer versions.

**Comment by kvaps** [comment]
My bad, 1.1.2 have no this issue.
